### PR TITLE
Create survey

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
We want to be able to add surveys to RM.

# What has changed
Created the full stack of Response Operations: UI & API.

# How to test?
Build using `make build` then start it using docker dev `make up` then navigate to http://localhost:7777

# Links
Trello: https://trello.com/c/0aP9EBOU

# Screenshots (if appropriate):
Screenshot 2021-07-26 at 17 31 43
Screenshot 2021-07-26 at 17 31 58
Screenshot 2021-07-26 at 17 32 22
Screenshot 2021-07-26 at 17 32 31
Screenshot 2021-07-26 at 17 32 41
Screenshot 2021-07-26 at 17 32 50

### CODE REVIEW

**Comment by NickGrantONS** [comment]
[ROps UI - first feature - create survey - PART ONE (13)](https://trello.com/c/0aP9EBOU/2504-rops-ui-first-feature-create-survey-part-one-13)

**Review by ryangrundy7 — APPROVED** [review]
Everything looks good and I can create surveys in the UI 👍 Might need Daves help for the travis error as I think it needs docker credentials for Travis

File: src/main/java/uk/gov/ons/ssdc/responseoperations/endpoint/AuthorisationEndpoint.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    this.userRepository = userRepository;
+  }
+
+  @GetMapping
+  public Set<UserGroupAuthorisedActivityType> getAuthorisedActivities(
+      @RequestParam(required = false, value = "surveyId") Optional<UUID> surveyId,
+      @Value("#{request.getAttribute('userEmail')}") String userEmail) {
+
+    // TODO: Remove this before releasing to production!
+    if (userEmail.equals("dummy@fake-email.com")) {
```
Comment (AdamHawtin) [inline src/main/java/uk/gov/ons/ssdc/responseoperations/endpoint/AuthorisationEndpoint.java:37]: Worth getting a card on the board for removing these/productionizing.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.godaddy</groupId>
+      <artifactId>logging</artifactId>
+      <version>1.2.5</version>
+    </dependency>
+    <dependency>
+      <groupId>com.vladmihalcea</groupId>
+      <artifactId>hibernate-types-5</artifactId>
```
Comment (AdamHawtin) [inline pom.xml:59]: Worth updating this out the gate?

**Review by AdamHawtin — APPROVED** [review]
Looks good and browser navigation all works as expected. 👍 

Couple of small non-blocking comments.

**Review by AdamHawtin — COMMENTED** [review]
Could use some README instructions for building/running locally.

**Pull request merged**